src/Components/ViewOpportunity/index.js

```diff
@@ -58,7 +59,7 @@ export function ViewOpportunity(props) {
             marginBottom: '200px',
           }}
         >
-          No opportunities, sorry!
+          Loading Opportunities if any...
         </h3>
       </div>
     );
```

Comment:
now, what if there are no opportunities at all? would not the users be waiting forever for some non-existent opportunities to be loaded? (which obviously would not load) 

what you could do is, 
- first, check if the `data` variable is `undefined`. If yes, show the loader to the user. 
- if the `data` variable is not `undefined` and is an array of length 0 `(data && Array.isArray(data) && data.length === 0)`, you can show that `no opportunity found`

Reply:
I have made the necessary changes